svc-aggregation/README.md

```diff
@@ -504,10 +527,8 @@ Content-Length:491 bytes
 
 ```
 
-
 >**Sample response header \(HTTP 201 status\)**
 
-
 ```
 "cache-control":"no-cache
```

Comment:
add system id in the location  of the response header

Reply:
Added.